app/controllers/admin/edition_workflow_controller.rb

```diff
@@ -78,6 +78,8 @@ def confirm_force_publish
     unless @edition.valid?(:publish)
       redirect_to admin_edition_path(@edition), alert: @edition.errors[:base].join(". ")
```

Comment:
It looks like a couple of tests are failing

I think for this one we just need to add a test which gives the user the next release permission and check it renders confirm_publish and update this test to confirm_published_legacy

<img width="1338" alt="image" src="https://user-images.githubusercontent.com/42515961/201624048-534487c0-cf78-4424-b810-0c22c9be6893.png">

And for this one 
<img width="1411" alt="image" src="https://user-images.githubusercontent.com/42515961/201623939-ba210717-a848-470d-a000-8ee778c5a315.png">

What's happening is it's checking for a redirect, but fails as there's a render action after the redirect

You can fix it by doing 

```
unless @edition.valid?(:publish)
   redirect_to admin_edition_path(@edition), alert: @edition.errors[:base].join(". ")
   return
end

render_design_system(:confirm_force_publish, :confirm_force_publish_legacy, next_release: true)
```
or there's a handy bit of rails syntax 

```
redirect_to admin_edition_path(@edition), alert: @edition.errors[:base].join(". ") and return unless @edition.valid?(:publish)

```

Reply:
All done, thank you!